api-extractor/src/definitions/ApiItem.ts

```diff
@@ -107,6 +107,19 @@ export interface IApiItemOptions {
 }
 
 /**
+ * Attributes of an import reference.
+ * 
+ * Example: 
+ * "import { MyClass as MyClass2, MyEnum } from '@ms/blah'", this is formatted in
+ * the interface as: 
+ * {packageName: "@ms/blah", exportedName "MyClass"}
+ */
+interface IImportTypeReference {
+  packageName: string;
+  exportedName: string;
+}
```

Comment:
You already declared something like this for ApiDefinitionReference.  When you get to the next step (checking \@public  functions that return \@internal types) you will need to do everything using the ApiDefinitionReference abstraction anyway.